conf/routes

```diff
@@ -12,8 +12,6 @@ GET	/docs/*file	controllers.Assets.at(path="/public/lib/swagger-ui/", file)
 # GET     /locations/json       controllers.ApplicationController.index
 
 
-GET	/patient/:socialId	controllers.EHealthVDCController.getPatientDetails(socialId: String)
-GET /patient/:socialId/blood-test/summary   controllers.EHealthVDCController.getAllTestValues(socialId: String)
-GET /patient/:socialId/blood-test/component/:testType controllers.EHealthVDCController.getTestValues(socialId: String, testType: String)
+GET /patient/:socialId/requesterId/:requesterId/blood-test/component/:testType controllers.EHealthVDCController.getAllValuesForBloodTestComponent(socialId: String, requesterId: String, testType: String)
```

Comment:
RequesterId  NOT part of the REST path.
Both requesterId and Purpose are sent in request headers. See https://stackoverflow.com/questions/14873927/play-framework-examine-http-request.